socorro/middleware/middleware_app.py

```diff
@@ -76,6 +76,8 @@
     (r'/suspicious/(.*)', 'suspicious.SuspiciousCrashSignatures'),
     (r'/util/(versions_info)/(.*)', 'util.Util'),
     (r'/adi/(.*)', 'adi.ADI'),
+    # the legacy one
```

Comment:
In fact, there's nothing to say really. Nothing of value. To say that it's a legacy hack, saying that belongs in the doc string of the implementation class.